pkgs/tools/inputmethods/fcitx5/fcitx5-anthy.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchurl, cmake, extra-cmake-modules, pkg-config, fcitx5, anthy, gettext }:
+
+stdenv.mkDerivation rec {
+  pname = "fcitx5-anthy";
+  version = "5.0.13";
+
+  src = fetchurl {
+    url = "https://download.fcitx-im.org/fcitx5/fcitx5-anthy/${pname}-${version}.tar.xz";
+    sha256 = "sha256-EGpyYiNalm4En3Oa++hyOSPLhjbkVcDyZt3LqamfSOI=";
+  };
+
+  nativeBuildInputs = [ cmake extra-cmake-modules pkg-config ];
+  buildInputs = [ fcitx5 anthy gettext ];
+
+  preInstall = ''
+    substituteInPlace src/cmake_install.cmake \
+      --replace ${fcitx5} $out
```

Comment:
It builds and works just fine both with and without it for me. It was there to begin with because I was initially basing this on [the derivation of fcitx-anthy](https://github.com/NixOS/nixpkgs/blob/2f9fd351ec37f5d479556cd48be4ca340da59b8f/pkgs/tools/inputmethods/fcitx-engines/fcitx-anthy/default.nix#L15-L18) which does the replacement for some reason.

Reply:
If it works without it, then it's probably best to remove it and the whole `preInstall` block then